### PR TITLE
Added bostrom network

### PR DESCRIPTION
**Pull request by SaveTheAles**

### CODE REVIEW

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -656,6 +656,34 @@
...
+        {
+          "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
+          "exponent": 0,
+          "aliases": [
+            "BOOT"
+          ]
+        },
+        {
+          "denom": "boot",
+          "exponent": 0
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:672]: BOOT asset have 0 exponent ? Can you confirme please, thanks

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -656,6 +656,34 @@
...
+      "name": "Bostrom",
+      "display": "boot",
+      "symbol": "BOOT",
+      "ibc": {
+        "source_channel": "channel-2",
+        "dst_channel": "channel-95",
+        "source_denom": "boot"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/boot.png"
```
Comment (IbrarMakaveli) [inline osmosis-1/osmosis-1.assetlist.json:685]: Can you change name image : bostrom.png -> boot.png
Thanks

**Review by IbrarMakaveli — CHANGES_REQUESTED** [review]

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (SaveTheAles) [reply osmosis-1/osmosis-1.assetlist.json:672]: Confirmed

File: osmosis-1/osmosis-1.assetlist.json (hunk 2 above)
Comment (SaveTheAles) [reply osmosis-1/osmosis-1.assetlist.json:685]: Done

File: osmosis-1/osmosis-1.assetlist.json (hunk 1 above)
Comment (sunnya97) [reply osmosis-1/osmosis-1.assetlist.json:672]: Hmm, I feel we should not have two `denom_units` entries with the same exponent.  It should just be listed as an alias instead.

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -656,6 +656,34 @@
...
+          "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
+          "exponent": 0,
+          "aliases": [
+            "BOOT"
+          ]
+        },
+        {
+          "denom": "boot",
+          "exponent": 0
+        }
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:673]:
```suggestion
        {
          "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4",
          "exponent": 0,
          "aliases": [
            "boot"
          ]
        }
```

**Review by IbrarMakaveli — APPROVED** [review]

**Pull request merged**